selftests/unit/test_archive.py

```diff
@@ -93,6 +94,8 @@ def test_tbz2_2_dir(self):
     def test_tbz2_2_file(self):
         self.compress_and_check_file('.tbz2')
 
+    @unittest.skipIf(sys.platform.startswith('darwin'),
+                     'macOS does not support archive extra attributes')
```

Comment:
Yep, this one makes sense to me as there is probably no easy way of determining this feature at the execution time.